modules/extensions/process/publisher/process/themes/default/partials/add_flowchart.hbs

```diff
@@ -0,0 +1,63 @@
+    <button class="btn btn-primary" id="flowchartSaveBtn" onclick="saveFlowchart()">Save
+    </button>
+    <button class="btn btn-primary" id="saveAsPNGBtn">Download As PNG</button>
+    <br/><br/>
+
+    <div style="width:100%; height: 10%; white-space:nowrap;">
+                            <span style="display: inline-block; vertical-align: top; padding: 5px; width:15%">
+                                <div id="myPalette" style="border: solid 1px gray; height: 602px">
+                                    <div class="jtk-demo-main" style="margin-top: 20px;">
+                                        <div class="jtk-demo-canvas canvas-wide flowchart-demo" id="canvas1"
+                                             style="border: none">
+                                            <div class="window start jsplumb-connected" id="startEv"
+                                                 title="To start the flowchart"><strong>
+                                                <p>start</p></strong></div>
+                                            <div style="border: 1px solid lightgray; width: 90%; top: 8em; position: absolute; left: 0.5em;"></div>
+                                            <div class="window step jsplumb-connected-step" id="stepEv"
+                                                 title="To describe a step"><strong>
+                                                <p>step</p></strong></div>
+                                            <div style="border: 1px solid lightgray; width: 90%; top: 15.25em; position: absolute; left: 0.5em;"></div>
+                                            <div class="window diamond jsplumb-connected-step" id="descEv"
+                                                 title="To describe a decision">
+                                                <strong><p class="desc-text">decision</p></strong></div>
+                                            <div style="border: 1px solid lightgray; width: 90%; top: 24.5em; position: absolute; left: 0.5em;"></div>
+                                            <div class="window step parallelogram jsplumb-connected-step" id="inpEv"
+                                                 title="To take/give input/output"><strong><p class="input-text">i/o</p></strong></div>
+                                            <div style="border: 1px solid lightgray; width: 90%; top: 31.5em; position: absolute; left: 0.5em;"></div>
+                                            <div class="window end jsplumb-connected-end" id="endEv"
+                                                 title="To end the flowchart"><strong>
+                                                <p>end</p></strong></div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </span>
+
+                            <span style="display: inline-block; vertical-align: top; padding: 5px; width:85%">
```

Comment:
This is fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7